lgr/tools/utils.py

```diff
@@ -303,6 +304,12 @@ def get_rz_label_script(label: str, unidb: UnicodeDatabase) -> str | None:
             script = unidb.get_script(character, alpha4=True)
             if script not in [COMMON_SCRIPT, INHERITED_SCRIPT]:
                 break
+        if script in [COMMON_SCRIPT, INHERITED_SCRIPT] and any(character in label for character in ['〆','ー']):
```

Comment:
I'm suggesting this but it's also not easy to read so you may not change it:
when breaking in python for loop, you may put a else statement after the for that you enter only if you never break